docs/radar.js

```diff
@@ -272,12 +273,14 @@ function radar_visualization(config) {
     }
   }
 
-  function legend_transform(quadrant, ring, legendColumnWidth, index=null) {
-    var dx = ring < 2 ? 0 : legendColumnWidth;
-    var dy = (index == null ? -16 : index * 12);
+  function legendTransform(quadrant, ring, legendColumnWidth, index=null, previousHeight = null) {
```

Comment:
What's the reason for changing the function name? Wouldn't this be a breaking change?
Let's revert to its prior name.

Reply:
I changed it because I see some function have camel case (the most recent one added) while some other (like this one) have snake case. Maybe we can find a standard and refactor in another pr all the functions name?
Anyway reverted for now to the previous name 🚀🚀🚀🚀

One last note: why do you say it is a breaking change? Because looking at the code the function here above is not accessible outside the `radar_visualization` one, that is basically the only public api of the tech radar. Am I missing something?